vmServer/send/proto/read_request.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python
```

Comment:
change the name of the file as print_request.py since it is just priniting rthe request? Also would recommend adding these scripts useful for debugging in a separate folder like debug

Reply:
Change has been made